### PR TITLE
fix expose ratelimit image.

### PR DESCRIPTION
**Pull request by qicz**
support #1296 

resolve #1203 review case

cc @arkodg @Xunzhuo

### CODE REVIEW

**Review by Xunzhuo — DISMISSED** [review]

**Comment by arkodg** [comment]
does this need to go into 0.4.0 ?
cc @AliceProxy

**Comment by Xunzhuo** [comment]
I think this is a follow-up of #1203 and #1218, we probably need it in v0.4.0

**Review by Xunzhuo — APPROVED** [review]

File: api/config/v1alpha1/shared_types.go [hunk 1]
```diff
@@ -8,14 +8,16 @@ package v1alpha1
...
+	// DefaultDeploymentReplicas is the default number of deployment replicas.
+	DefaultDeploymentReplicas = 1
 	// DefaultDeploymentCPUResourceRequests for deployment cpu resource
 	DefaultDeploymentCPUResourceRequests = "100m"
 	// DefaultDeploymentMemoryResourceRequests for deployment memory resource
 	DefaultDeploymentMemoryResourceRequests = "512Mi"
 	// DefaultEnvoyProxyImage is the default image used by envoyproxy
 	DefaultEnvoyProxyImage = "envoyproxy/envoy-dev:latest"
+	// DefaultRateLimitImage is the default image used by ratelimit.
+	DefaultRateLimitImage = "envoyproxy/ratelimit:f28024e3"
```
Comment (kflynn) [inline api/config/v1alpha1/shared_types.go:20]: I kind of feel like either this shouldn't be pinned, or the Envoy version should be -- having one pinned to a specific version and the other not feels weird to me. How will this get updated?

File: docs/latest/user/rate-limit.md [hunk 2]
```diff
@@ -143,6 +143,7 @@ data:
         rateLimitDeployment:
           replicas: 1
           container:
+            image: envoyproxy/ratelimit:latest
```
Comment (kflynn) [inline docs/latest/user/rate-limit.md:146]: This makes me feel especially weird about the default being pinned. 🙂

**Review by kflynn — CHANGES_REQUESTED** [review]
Looks fine except that having the default ratelimit image pinned to a specific version feels a little odd to me?

File: api/config/v1alpha1/shared_types.go (hunk 1 above)
Comment (qicz) [reply api/config/v1alpha1/shared_types.go:20]: ref  ratelimit_deployment.go

**Comment by qicz** [comment]
> Looks fine except that having the default ratelimit image pinned to a specific version feels a little odd to me?

`Default` image just eg initial config, can change this by `RateLimitDeployment`

**Pull request merged**